clients/ops/admin-ui/src/constants.ts

```diff
@@ -25,6 +25,14 @@ export const USER_PRIVILEGES: UserPrivileges[] = [
     privilege: "Create or Update datastore connections",
     scope: "connection:create_or_update",
   },
+  {
+    privilege: "Instantiate connections to SaaS datastores",
+    scope: "connection:instantiate",
+  },
+  {
+    privilege: "Read connection types",
+    scope: "connection_type:read",
+  },
```

Comment:
I've added these as a way to assign the necessary permissions for creating and configuring SaaS connectors to new users.